src/Device/deviceInfo/deviceInfo.jsx

```diff
@@ -110,14 +155,38 @@ const deviceInfo = pcaNumber => ({
     pca20035: {
```

Comment:
I suggest to use all caps 'PCA20035' and sanitise the pca key, better safe than sorry when a firmware update comes.

Reply:
Ah, yes, I remember that you already commented that on the other PR which I did not yet get around to implement. Implemented now here in 261b9bd.